src/Utils/ExplicitPagination.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/*
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+declare(strict_types=1);
+
+namespace Google\Generator\Utils;
+
+class ExplicitPagination
+{
+    /**
+     * This is a list of explicit pagination for the pagination heuristics.
+     * If there is a proto that has multiple repeated fields and we want to
+     * manually override which of the repeated fields is the one should be paginated on
+     * should be added to this list
+     */
+    private static array $paginations = [
+        'Google.Cloud.Compute.V1.UsableSubnetworksAggregatedList' => 'items',
```

Comment:
I was assuming this would be the namespace, e.g. `Google\Cloud\Compute\V1\UsableSubnetworksAggregatedList`.  However, what you have here also works... as long as it's correct. In this case, it should be lowercase, to [match the package name](https://raw.githubusercontent.com/googleapis/googleapis/refs/heads/master/google/cloud/compute/v1/compute.proto)
```suggestion
        'google.cloud.compute.v1.UsableSubnetworksAggregatedList' => 'items',
```